src/main/scala/org/broadinstitute/dsde/firecloud/service/MethodsService.scala

```diff
@@ -1,60 +1,35 @@
 package org.broadinstitute.dsde.firecloud.service
 
-import akka.actor.{Actor, _}
-import com.wordnik.swagger.annotations.{Api, _}
-import org.broadinstitute.dsde.firecloud.MethodsClient
-import org.broadinstitute.dsde.firecloud.model.MethodEntity
+import akka.actor.{Actor, Props}
+import com.wordnik.swagger.annotations.{Api, ApiOperation}
 import org.slf4j.LoggerFactory
+import spray.client.pipelining._
 import spray.routing.{HttpService, Route}
 
+import org.broadinstitute.dsde.firecloud.{FireCloudConfig, HttpClient}
+
 class MethodsServiceActor extends Actor with MethodsService {
   def actorRefFactory = context
   def receive = runRoute(routes)
 }
 
-@Api(value = "/methods", description = "Methods Service", produces = "application/json")
+@Api(value = "/methods", description = "Methods Service", produces = "application/json, text/plain")
```

Comment:
Even in error conditions? Are there any servers in front of Agora that could potentially return text in an error condition? Or maybe I'm looking at this the wrong way in that it doesn't have to be correct all the time just most of the time.